### PR TITLE
create registered reports author guide

### PR DESCRIPTION
**Pull request by neilernst**
This is the draft MSR CFP - we will use this for discussion more broadly prior to release.

### CODE REVIEW

File: msr_rr_cfp.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## Background/Context
+What is your research about? Why are you doing this research, why is it interesting?
+
+**Example**: The enterprise is the flag ship of the federation, and it allows families to travel onboard. However, there are no studies that evaluate how this affects the crew members”
+
+## Objective/Aim
+What exactly are you evaluating? What are the objects of the study?
+
+**Example**: We are studying whether the frequency of sick days, the work effectiveness and efficiency differ between science officers who bring their family with them, compared to science officers who are serving without their family
+
```
Comment (robertfeldt) [inline msr_rr_cfp.md]: Might this need to be a bit "beefier" for a RR? I.e. given that the specific RQs / hypothesis are in focus (and should not later be changed) and there is less to say about results (since there are little to no) it makes sense to expect more of the Objective/Aim section? Are there existing RR's from other fields we can use as models/examples?

File: msr_rr_cfp.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+## Method
+How are you addressing your objective?
+**Example**: We conduct an observational study and use a between subject design. To analyze the data, we use a t test or Wilcoxon test, depending on the underlying distribution
+
```
Comment (robertfeldt) [inline msr_rr_cfp.md]: Maybe we should expect also some characterisation of data sources?

File: msr_rr_cfp.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+## Early Results
+Here come the pilot data, if you have any
+
+## Limitations
+
+# Introduction (optional)
```
Comment (robertfeldt) [inline msr_rr_cfp.md]: Why make this optional? Seems it is always needed to give context and motivation?

File: msr_rr_cfp.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+Give more details on the bigger picture of your study and how it contributes to this bigger picture
+
+## Hypotheses / research questions (required)
+Clearly state the research hypotheses or research questions that you want to test with your study, and a rationalization for the hypotheses/questions
+
+**Example**: Science officers with their family on board have more sick days than science officers without their family
+
+Rationale: Since toddlers are often sick, we can expect that crew members with their family onboard need to take sick days more often. 
+
+If you do not have a hypothesis/question, please elaborate why (e.g., we follow a grounded theory approach, in which we generate and refine the hypotheses/questions as we collect data)
```
Comment (robertfeldt) [inline msr_rr_cfp.md]: But what would it mean to do an RR for a explorative/GT type of study? I think we need to link to examples of such RRs as well as final papers.

File: msr_rr_cfp.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+**Example**: Each crew member needs to sign the informed consent and agreement to process his or her data according to GDPR. Then, we conduct the interviews. Afterwards, participants need to complete the simulated task. 
+
+# Analysis Plan (required)
+## Descriptive statistics
+How do you describe the data? How do you handle outliers?
+
+Example: To represent the number of sickdays, we use histograms. Dependending on the distribution, we remove values that are 2 standard deviations above the mean as outliers (normal distribution). If the data are non-normal, we use the median and values below the 10th/above the 90th percentile.
+
+## How do you evaluate the hypotheses?
+Are you using significance tests? How are you avoiding inflation of type-I error (if applicable)
```
Comment (robertfeldt) [inline msr_rr_cfp.md]: I'd like this to be written in a way that also more advanced (IMHO) statistical procedures (Bayesian, for example) are also allowed/ok and so that we don't imply a bias for classical, Fisherian, parametric analysis methods/approaches.

**Comment by robertfeldt** [comment]
I've added comments inline in the cfp. Overall looks good but I do think it will be crucial to identify and explicitly link to model/example RRs and final papers from other disciplines (if we cannot find SE ones). Good examples are probably more important than detailed spec here imho.

**Comment by robertfeldt** [comment]
Also I didn't quite get why this is the CfP? Focus was more on expected format of submissions? Shouldn't we rather link to that format description from the CfP? Maybe I missed some other files... ;)

File: msr_rr_cfp.md (hunk 4 above)
Comment (neilernst) [reply msr_rr_cfp.md]: True. It might make sense to leave exploratory out altogether for now. We could iterate with ICSME or other RR initiatives

File: msr_rr_cfp.md (hunk 5 above)
Comment (neilernst) [reply msr_rr_cfp.md]: :) Maybe better to say less here. In general the OSF guide is pretty vague, while we are very detailed - people might get confused.

**Comment by neilernst** [comment]
Yeah, we weren't sure if this is a CFP or author's guide. A bit of both.

File: msr_rr_cfp.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+**Example**: To test for normality, we use a Shapiro-Wilk test. For efficiency, we use a t test / Wilcoxon test depending on the distribution. To evaluate the effect of gender, we use a two-way ANOVA / Friedman test, depending on the distribution.
+
+# Examples
```
Comment (neilernst) [inline msr_rr_cfp.md]: we could do with some more SE specific examples, but I could not find any.

Comment (torkar) [reply msr_rr_cfp.md]: I think this is way too specific and, in addition, very one-sided focusing on frequentist language.

File: msr_rr_cfp.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+What exactly are you evaluating? What are the objects of the study?
+
+**Example**: We are studying whether the frequency of sick days, the work effectiveness and efficiency differ between science officers who bring their family with them, compared to science officers who are serving without their family
+
+## Method
+How are you addressing your objective? What data sources are you using.
+**Example**: We conduct an observational study and use a between subject design. To analyze the data, we use a t test or Wilcoxon test, depending on the underlying distribution. Our data come computer monitoring of Enterprise crew members.
+
+## Early Results
+Here come the pilot data, if you have any. Give the reviewer some indication that you have reason to believe this research plan is promising.
```
Comment (ctreude) [inline msr_rr_cfp.md]: I'm a bit worried that "Early Results" suggests that this is just another NIER / ERA track. Is this really needed, and if so, would it make sense to make it very explicit that this is not needed for acceptance?

File: msr_rr_cfp.md (hunk 6 above)
Comment (neilernst) [reply msr_rr_cfp.md]: Is your paper a suitable reference or perhaps has language we could borrow? The maturity model one?

File: msr_rr_cfp.md (hunk 7 above)
Comment (neilernst) [reply msr_rr_cfp.md]: maybe "preliminary results"? the purpose is to show a proof of concept/plan. 

(Incidentally my feeling is that this is exactly what a NIER paper should be)

File: msr_rr_cfp.md (hunk 6 above)
Comment (torkar) [reply msr_rr_cfp.md]: Would it be possible to provide two examples: One as above and the other focusing on BDA? Then we provide two references, one could be the maturity paper, and one could be one of the Bayesian papers we've started to push out? Providing Gelman's BDA book will probably be a bit over the top...

Comment (torkar) [reply msr_rr_cfp.md]: Also, we should not prohibit people to deviate from the plan of analysis. They, however, need to provide solid reasons as to why they deviated from the plan.

File: msr_rr_cfp.md (hunk 7 above)
Comment (ctreude) [reply msr_rr_cfp.md]: Right. And to de-emphasise this aspect, I'd probably suggest to not even mention results in the CfP. Not sure about MSR, but at ICSME we'll have NIER and RR -- the lack of results and 100% focus on methodology for RR could be what sets the two apart.

File: msr_rr_cfp.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
+# Title (required)
+Provide the working title of your study. It may be the same title that you submit for publication of your final manuscript, but it is not a requirement.
+* **Example**:: Should your family travel with you on the enterprise?
+* Subtitle (optional): Effect of accompanying families on the work habits of crew members
+
+# Authors (required)
+At this stage, we believe that an unblinded/single blind review is most productive
```
Comment (dgraziotin) [inline msr_rr_cfp.md]: How would this play out if the final study is then submitted to a DBR venue? I guess that a registered report is going to be a public document.

File: msr_rr_cfp.md (hunk 1 above)
Comment (dgraziotin) [reply msr_rr_cfp.md]: How much sense would it make for a RR to _not_ be hypothesis-based? 
Isn't this what would differentiate a RR from a NIER/short paper on "an idea/a proposed study"?

File: msr_rr_cfp.md (hunk 8 above)
Comment (neilernst) [reply msr_rr_cfp.md]: well in this case, it will go to EMSE, which is single blind. But the RR format on open science foundation does have an embargo option.

File: msr_rr_cfp.md (hunk 7 above)
Comment (neilernst) [reply msr_rr_cfp.md]: ok, we can probably handle cases where the reviewer is unconvinced by the potential of the study in the rebuttal phase, where preliminary results could be used.

File: msr_rr_cfp.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+## Early Results
+Here come the pilot data, if you have any
+
+## Limitations
+
+# Introduction (optional)
+Give more details on the bigger picture of your study and how it contributes to this bigger picture
+
+## Hypotheses / research questions (required)
+Clearly state the research hypotheses or research questions that you want to test with your study, and a rationalization for the hypotheses/questions
```
Comment (dgraziotin) [inline msr_rr_cfp.md]: How much sense would a RR of a research question be? As opposed to stating hypotheses. I think my doubt here is similar to the one of @robertfeldt below on exploratory studies.

File: msr_rr_cfp.md (hunk 5 above)
Comment (dgraziotin) [reply msr_rr_cfp.md]: How about being more general so that other statistical procedures are implied? Something like "How are you testing the significance of your results?"

File: msr_rr_cfp.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+# Analysis Plan (required)
+## Descriptive statistics
+How do you describe the data? How do you handle outliers?
+
+Example: To represent the number of sickdays, we use histograms. Dependending on the distribution, we remove values that are 2 standard deviations above the mean as outliers (normal distribution). If the data are non-normal, we use the median and values below the 10th/above the 90th percentile.
+
+## How do you evaluate the hypotheses?
+Are you using significance tests? How are you avoiding inflation of type-I error (if applicable)
+
+**Example**: To test for normality, we use a Shapiro-Wilk test. For efficiency, we use a t test / Wilcoxon test depending on the distribution. To evaluate the effect of gender, we use a two-way ANOVA / Friedman test, depending on the distribution.
```
Comment (dgraziotin) [inline msr_rr_cfp.md]: To test for non-normality? I am fun at parties 🥳

File: msr_rr_cfp.md (hunk 7 above)
Comment (ctreude) [reply msr_rr_cfp.md]: And pick reviewers who don't expect results in an RR submission :)

File: msr_rr_cfp.md (hunk 1 above)
Comment (mendezfe) [reply msr_rr_cfp.md]: Might be also too much, but maybe we can slightly rephrase it to make explicit that both confirmatory and exploratory types of studies (to name just two) are welcome? Maybe we can add a curiosity-driven study as one example for having hypothesis as an actual outcome of the study and one where we want to test certain hypothesis? If you agree, I would then propose to change the term "evaluate" to "study" or "investigate" (or something similarly vague). Would then also be consistent with the following parts.

File: msr_rr_cfp.md (hunk 10 above)
Comment (mendezfe) [reply msr_rr_cfp.md]: That is actually a hypothesis I can easily reject. ;-)

File: msr_rr_cfp.md (hunk 7 above)
Comment (nnovielli) [reply msr_rr_cfp.md]: I also think that (even early) results should not be expected in an RR submission. I would expect an RR reviewer to mainly focus on assessing the soundness of the hypothesis/RQs and the soundness of the proposed methodology. While early results in NIER papers are useful to strengthen the authors' vision, i.e. by demonstrating feasibility of the new ideas, potential impact,  etc., I think one of the added values of the RR track is to have the possibility to get a feedback on the methodology and study design itself by mitigating the reviewers' bias that could emerge due to, for example, negative results.

File: msr_rr_cfp.md (hunk 1 above)
Comment (neilernst) [reply msr_rr_cfp.md]: Given the NIER tracks, I suggest we direct curiosity-driven research to those tracks instead of as RR, at least for now. Thus RR would only handle confirmatory studies.

File: msr_rr_cfp.md (hunk 7 above)
Comment (neilernst) [reply msr_rr_cfp.md]: remove early results section, see fc825ffc2a3a0185564e900e0d726153f3d64f73

File: msr_rr_cfp.md (hunk 6 above)
Comment (neilernst) [reply msr_rr_cfp.md]: deviation part is managed on the sibling PR: #21

File: msr_rr_cfp.md (hunk 9 above)
Comment (neilernst) [reply msr_rr_cfp.md]: remove RQ

File: msr_rr_cfp.md (hunk 6 above)
Comment (neilernst) [reply msr_rr_cfp.md]: added Bayesian language + ref in latest

File: msr_rr_cfp.md (hunk 1 above)
Comment (torkar) [reply msr_rr_cfp.md]: I would find it a bit peculiar that we do not allow RRs for exploratory research. I don't want to be a party pooper but why don't we use a more general, and more "established" template, and allow reviewers the possibility to provide input without us being too prescriptive? https://github.com/crsh/prereg

Comment (neilernst) [reply msr_rr_cfp.md]: Yeah, to some extent we don't know what we will get, so might be easier to be "liberal in what we accept" to use Postel's law. 

However we have had some good comments about overlap with NIER tracks. Do you not see a conflict with NIER?

Comment (torkar) [reply msr_rr_cfp.md]: Well, honestly, I don't care about NIER. I care more that we get this right. Disqualifying exploratory research seems wrong imho.

Comment (neilernst) [reply msr_rr_cfp.md]: fixed this in 46b307c10f68b2df1a374c

File: registered-reports_guide.md [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+Clearly state the research hypotheses that you want to test with your study, and a rationalization for the hypotheses. 
+
+* **Example**: Science officers with their family on board have more sick days than science officers without their family
+
+Rationale: Since toddlers are often sick, we can expect that crew members with their family onboard need to take sick days more often. 
+
+For now, we focus the registered reports track on confirmatory research, ie. where a well-established hypothesis can be formulated. While pre-registration of exploratory studies may be possible, at this time we focus on the confirmatory reports. The NIER track is a good venue for exploratory/emerging results.
+
+# Introduction
+Give more details on the bigger picture of your study and how it contributes to this bigger picture. An important component pf pHase 1 review is assessing the novelty and relevance of the study questions, so be sure to explain this.
```
Comment (ctreude) [inline registered-reports_guide.md]: I would suggest to omit "novelty" here to clearly distinguish from NIER.

Comment (neilernst) [reply registered-reports_guide.md]: thanks, I've updated this in the latest patch

**Comment by aserebrenik** [comment]
I like the idea, but the example puts too much stress on gender-binary. Can we solve this somehow?

**Comment by neilernst** [comment]
@aserebrenik good point, yes, I'll fix the examples - thanks.

**Comment by neilernst** [comment]
the latest commit has removed gender references  in favour of Alien Species as blocking variable - the benefits of Star Trek examples. @aserebrenik - let me know if I missed something.

**Comment by monperrus** [comment]
Thanks for the registered reports author guide!
